src/content/blog/content-layer-deep-dive.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: 'Content Layer: A Deep Dive'
+description: There's a new way to handle content in Astro. Take a deep dive into the Content Layer API, and learn how it can help you build even more kinds of sites using Astro.
+homepageLink:
+  title: "Content Layer"
+  subtitle: New experimental Content Layer API
+publishDate: '2024-09-18'
+authors:
+  - matt
+lang: 'en'
+coverImage: "/src/content/blog/_images/content-layer-deep-dive/blog-post-content-layer.png"
+socialImage: "/src/content/blog/_images/content-layer-deep-dive/og-content-layer.png"
+---
+
+import BlogContentImage from '/src/components/BlogContentImage.astro';
+import BlogSummary from '/src/components/BlogSummary.astro';
+import astroBuildImage from '/src/content/blog/_images/content-layer-deep-dive/astro-build.png';
+import contentLayerArchitectureImage from '/src/content/blog/_images/content-layer-deep-dive/content-layer-architecture.png';
+
+<BlogSummary>
+	This week we have released the first beta of Astro 5, which includes a whole new way to handle content in Astro. This post takes a deep dive into the Content Layer API, showing how it works and how you can use it to build your sites.
+</BlogSummary>
+
+Astro was born to create content-driven websites. While it can now be used to build all sorts of dynamic apps too, it is still the very best for sites that are built around lots of content. From full-featured docs sites built with Starlight like [Cloudflare](https://developers.cloudflare.com/) and [StackBlitz](https://tutorialkit.dev/), to beautiful marketing sites for brands such as [Porsche](https://www.porsche.com/usa/) and [Netlify](https://www.netlify.com), millions of users every day are already appreciating the fast and accessible sites that you can build with Astro, and thousands of engineers love the best developer experience in the industry.
+
+In Astro 2 we introduced [Content Collections](https://docs.astro.build/en/guides/content-collections/) as a powerful new way to organize your local content, build with type safety, and scale to thousands of pages. Content Collections provide a best-in-class developer experience for local files such as Markdown and MDX, but we heard from you that you wanted the same benefits for all of your content including remote APIs. It was also clear that while many people were building sites with thousands of pages, our Content Collections API struggled to scale into the tens of thousands of pages, with slower builds and excessive memory usage.
```

Comment:
```suggestion
In Astro 2 we introduced [Content Collections](https://docs.astro.build/en/guides/content-collections/) as a powerful new way to organize your local content, build with type safety, and scale to thousands of pages. Content Collections provide a best-in-class developer experience for local files such as Markdown and MDX, but we heard from you that you wanted the same benefits for all of your content including remote APIs. It was also clear that while many people were building sites with thousands of pages, our Content Collections API struggled to scale into the many tens of thousands of pages, with slower builds and excessive memory usage.
```

adjust this if you want, just felt that "tens of thousands" makes it sound like 10k pages can't be built, which isn't true. Adding "many" or some other qualifiers makes this a little less firm on the number.

Reply:
I felt that the plural was enough there